### ISSUE TITLE
Insufficient color contrast

### ISSUE THREAD

**Issue by BarrierbreakApp** [google/recaptcha#430]
Example URL: https://www.google.com/recaptcha/api2/demo

Background/foreground color combinations with an insufficient contrast ratio are found:
• The #FFFFFF of "Select all squares with If there are none, click skip ", "skip" and "Verify" (that becomes available on activating "I'm not a robot" checkbox > Get an audio challenge button) has an insufficient color contrast ratio of 3.3:1 with a #4A90E2 background.
• The #FFFFFF  "If there are none, click skip" (that becomes available on activating "I'm not a robot" checkbox > images) has an insufficient color contrast ratio of 1.7:1 with a #A5C8F1 background.
• The #FF1B1B error message text "Multiple correct solutions required - please solve more." (that becomes available on activating "I'm not a robot" checkbox > Get an audio challenge" button) has a color contrast ratio of 3.9:1 with #FFFFFF background.

Suggestion:
On all pages, ensure that each text/background color combination presents a contrast ratio of at least 4.5:1 for small text and 3:1 for text that is at least 18 points or bold 14 points.

![reCaptcha_10](https://user-images.githubusercontent.com/16647640/107736337-d46d0a00-6d27-11eb-95b7-e1b01da25e55.png)

**Comment by rowan-m** [comment]
These all look like automated issues for the widget unrelated to the PHP client library in this repo.
